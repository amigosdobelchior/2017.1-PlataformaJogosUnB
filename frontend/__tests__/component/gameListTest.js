import React from 'react';
import {Grid} from "semantic-ui-react";
import renderer from 'react-test-renderer';
var expect = required('expect');

test('Test render GameList', () => {
  const component = renderer.create(
      <Grid doubling columns={5}>
          [{'game1'},{'game2'},{'game3'}]
      </Grid>
     );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
