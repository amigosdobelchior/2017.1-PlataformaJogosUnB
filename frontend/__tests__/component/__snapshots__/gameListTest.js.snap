// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render GameList 1`] = `
<div
  className="ui doubling five column grid"
>
  [
  game1
  ,
  game2
  ,
  game3
  ]
</div>
`;
